src/contact.rs

```diff
@@ -496,6 +506,16 @@ impl Contact {
             set_blocked(context, Nosync, contact_id, false).await?;
         }
 
+        if sync.into() {
+            chat::sync(
+                context,
+                chat::SyncId::ContactAddr(addr.to_string()),
+                chat::SyncAction::Rename(name.to_string()),
+            )
+            .await
+            .log_err(context)
+            .ok();
+        }
         Ok(contact_id)
     }
```

Comment:
why not do the `chat::sync()` call in `create() ` and make `create_ex` never sync? 
the only caller of `create_ex` uses `nosync` so this would simplify func signatures and logic.

Reply:
Not sure here. Walked through all such functions, in 4 of 8 of them this block can be moved out so that they can be renamed to `*_nosync()`, but maybe having a uniform API is better.